apps/api/src/app/topics/dtos/filter-topics.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiPropertyOptional } from '@nestjs/swagger';
+
+import { TopicDto } from './topic.dto';
+
+export class FilterTopicsRequestDto {
+  @ApiPropertyOptional({
+    type: Number,
+    required: false,
```

Comment:
Doesn't `ApiPropertyOptional` decorator already add the `required: false`?

Reply:
Good spot